src/enacrestic/__init__.py

```diff
@@ -619,7 +623,8 @@ def _start_when_systray_available(self):
 
         self.tray_icon.setContextMenu(menu)
 
-        self.state.connect_to_gui(self.tray_icon, info_action, autostart_action)
+        self.state.connect_to_gui(
```

Comment:
As this doesn't change anything, could it be removed from PR ?

Reply:
Done